pkg/endpoint/metadata/k8s_metadatafetcher.go

```diff
@@ -18,6 +18,12 @@ import (
 	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/labelsfilter"
 	"github.com/cilium/cilium/pkg/option"
+	"github.com/cilium/cilium/pkg/time"
+)
+
+const (
+	// Maximum amount of time to wait for pod or namespace to appear.
+	fetchWaitTime = 100 * time.Millisecond
```

Comment:
> The pod and namespace data is now stored in StateDB tables and by default the reflection into these tables waits 50ms before committing (for larger transactions and to avoid unnecessary intermediate states).

I don't know where these 50 ms are defined but can we export them and use 2 times that variable in here?

Reply:
I guess we could define it in e.g. `daemon/k8s/pods.go` or use the default wait time from `pkg/k8s/statedb.go`. Will look into it.